Tribler/Core/Upgrade/upgrade.py

```diff
@@ -20,10 +19,24 @@
 
 class TriblerUpgrader(object):
 
-    def __init__(self, session, db):
+    _singleton = None
```

Comment:
We don't want any more singletons, it makes things harder to test.

Just make each Session instance its own Upgrade object.